src/index.html

```diff
@@ -11,7 +11,203 @@
+
+        <input
+          id="date"
+          type="date"
+          name="dateOfBirth"
+          value="mm/dd/yyyy"
+          class="input"
+        >
+
+        <br>
+
+        <label for="checkbox">I accept the term of the agreement</label>
+
+        <input
+          id="checkbox"
+          type="checkbox"
+          name="terms"
+        >
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label for="email">E-mail:</label>
+
+        <input
+          id="email"
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          class="input"
+          required
+        >
+
+        <br>
+
+        <label for="password">Password:</label>
+
+        <input
+          id="password"
+          type="password"
+          name="password"
+          minlength="8"
+          maxlength="30"
+          required
+        >
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label>Do you love cats?</label>
+
+        <input
+          id="yes"
+          type="radio"
+          name="cat"
+          value="yes"
+          class="input"
+        >
+
+        <label for="yes" class="input">Yes</label>
+
+        <input
+          id="no"
+          type="radio"
+          name="cat"
+          value="No"
+          class="input"
+        >
+
+        <label for="no" class="input">No</label>
+
+        <br>
+
+        <label for="color">What is your favorite color?
+          <input
+            id="color"
+            type="color"
+            name="color"
+            class="input"
+          >
+        </label>
+
+        <br>
+
+        <label for="time">What time do you go to bed?</label>
+
+        <input
+          id="time"
+          type="time"
+          name="timeToBed"
+          step="2"
+          class="input"
+        >
+
+        <br>
+
+        <label for="car" class="input">
+          What are your favorite brand of cars?
+          <select
+          id="car"
+          name="carBrand"
+          multiple
+        >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <br>
+
+        <label for="rate">How do you rate our work?</label>
+
+        <input
+          id="rate"
+          type="range"
+          name="rate"
+          min="0"
+          max="100"
+          value="0"
+        >
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label for="comments" class="input">
+          Comments:
+          <textarea
+            id="comments"
+            name="txt"
+          ></textarea>
+        </label>
+
+        <br>
+
+        <label for="recommend" >Would you recommend us?</label>
+        <select
+          id="recommend"
+          name="recommend"
+        >
```

Comment:
Add empty line between siblings
```suggestion
        <label for="recommend" >Would you recommend us?</label>
        
        <select
          id="recommend"
          name="recommend"
        >
```